script.js

```diff
@@ -0,0 +1,23 @@
+const containMenu = document.getElementById('container_menu');
+const hamburgerMenu = document.getElementById('hamburger_menu');
+const closeSidebar = document.querySelector('.sidebar_menu');
+const closeButton = document.getElementById('close_button');
+const navElement = document.querySelectorAll('.menu_mobile a');
+
+hamburgerMenu.addEventListener('click', () => {
+  containMenu.style.display = 'flex';
+});
+
+closeSidebar.addEventListener('click', () => {
+  containMenu.style.display = 'none';
+});
+
+closeButton.addEventListener('click', () => {
+  containMenu.style.display = 'none';
```

Comment:
Kindly keep your code `D-R-Y`. This same code is repeated 3 times. This defeats the purpose of `OOP`. You could use this code as a function and call it in the `addEventListener` method as the second argument